terraform/eks/otlp.tf

```diff
@@ -291,6 +293,26 @@ resource "null_resource" "aoc_deployment_adot_operator" {
   depends_on = [module.adot_operator]
 }
 
+resource "kubectl_manifest" "java_auto_instrumentation_deployment" {
```

Comment:
Where does `kubectl_manifest` module come from? Who is the provider?

Why not use `kubernetes_manifest` in the official `kubernetes` provider? 
https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/manifest

Reply:
I was getting `kubernetes_manifest unmarshaling unknown values is not supported` error with the same yaml template with the `kubernetes_manifest`. 
`kubectl_manifest`:  https://registry.terraform.io/providers/cpanato/kubectl/latest/docs/resources/kubectl_manifest

Reply:
Updated accordingly with `kubernetes_manifest`.